### PR TITLE
Clarify that project only supports Python 3

### LINKED ISSUE #7: The readme should point out that this tool only works on python3.
the readme should point out that this tool only works on python3. I hit the exact same issue today.

_Originally posted by @samos123 in https://github.com/cisagov/travis-wait-improved/issues/5#issuecomment-572406586_

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

Explicitly state in README that Python 2 is not supported.
Add Python version requirement to `setup.py`
Merge in the latest upstream skeleton changes, the largest being a switch from Travis to GitHub Actions.
Bumping version to 1.0.0 since this is in production.

## 💭 Motivation and Context

We've had at least two people comment that it does not support python 2.  We don't want to
waste anyone's time trying to debug python versions.  This should help avoid that going forward.

- Addresses issues #5 #6 #7
- Resolves #7 

## 🧪 Testing

Ran tests in pytest, and GitHub Actions.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: bump_version.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# bump_version.sh (show|major|minor|patch|prerelease|build)
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+VERSION_FILE=src/example/_version.py
```
Comment (mcdonnnj) [inline bump_version.sh]: This should reflect the version file in this project.

**Review by mcdonnnj — CHANGES_REQUESTED** [review]
I'm digging making clear when projects don't support Python 2 with EoL⚰️ upon us. Do we still need the Travis configuration file with the addition of Github Actions to this repository? Other than that please see this one change. Strong start to 2020 💪

File: bump_version.sh (hunk 1 above)
Comment (dav3r) [reply bump_version.sh]: Good catch @mcdonnnj

**Review by dav3r — APPROVED** [review]
Looks good, aside from the comments from @mcdonnnj that should be taken care of (remove `.travis.yml` and point to correct `VERSION_FILE` in `bump_version.sh`). 🐍🐍🐍

**Review by jsf9k — CHANGES_REQUESTED** [review]
I have the same issues as @mcdonnnj.  Otherwise this looks like quality work to me!

File: bump_version.sh (hunk 1 above)
Comment (felddy) [reply bump_version.sh]: Fix `example` thangs in 6e4c3db
Removed Travis config.

**Review by jsf9k — APPROVED** [review]

**Review by mcdonnnj — APPROVED** [review]
I'm digging it! 👍🚀

**Pull request merged**